### PR TITLE
Fix all non-undefined PHPStan ignores 1/2

### PR DESCRIPTION
**Pull request by mvorisek**

### CODE REVIEW

**Comment by mvorisek** [comment]
@ibelar can you please fix the remaining PHPStan errors here?

File: phpstan.neon.dist [hunk 1]
```diff
@@ -64,31 +47,15 @@ parameters:
         - '~^Access to an undefined property atk4\\ui\\AbstractView\:\:\$cb\.$~'
         - '~^Call to an undefined method atk4\\ui\\AbstractView\:\:setModel\(\)\.$~'
         - '~^Call to an undefined method atk4\\ui\\AbstractView\:\:jsReload\(\)\.$~'
-        # for demos/_unit-test/scope-builder-to-query.php
-        - '~^Method atk4\\ui\\App\:\:decodeJson\(\) invoked with 2 parameters, 1 required\.$~'
         # for demos/basic/label.php
         - '~^Call to an undefined method atk4\\ui\\Jquery\:\:fadeOut\(\)\.$~'
```
Comment (mvorisek) [inline phpstan.neon.dist:51]: we can fix all `...ndefined method atk4\\ui\\Jquery...` errors easily, just find a list of Jquery functions and phpdoc them

1/2 of the remain error list will be solved

File: src/Jquery.php [hunk 2]
```diff
@@ -7,40 +7,73 @@
...
- * @method Jquery form($a=null)
- * @method Jquery api($a=null)
- * @method Jquery visibility()
+ * For SemanticUI:
+ * @method Jquery api(...$args)
+ * @method Jquery form(...$args)
+ * @method Jquery visibility(...$args)
+ *
+ * For Atk4UI:
+ * @method Jquery atkAjaxec()
```
Comment (mvorisek) [inline src/Jquery.php]: @ibelar please update virtual method phpdocs with real params

incl. string type, example:
```
@method Jquery atkExample(string $name, int $index)
```

**Review by georgehristov — APPROVED** [review]

File: src/Form/Control/Multiline.php [hunk 3]
```diff
@@ -691,7 +692,7 @@ protected function renderView(): void
         $this->cb->set(function () {
             try {
                 return $this->renderCallback();
-            } catch (\Atk4\Core\Exception | \Error $e) {
+            } catch (\atk4\Core\Exception | \Error $e) {
```
Comment (mvorisek) [inline src/Form/Control/Multiline.php]:
```suggestion
            } catch (\Atk4\Core\Exception | \Error $e) {
```

File: phpstan.neon.dist [hunk 4]
```diff
@@ -43,61 +30,29 @@ parameters:
...
         # for tsrc/Table/Column/FilterModel/TypeTime.php
         - '~^Call to an undefined method Atk4\\Ui\\Table\\Column\\FilterModel\\TypeTime\:\:recalData\(\)\.$~'
-        # for tsrc/View.php
-        - '~^Method Atk4\\Ui\\View\:\:setModel\(\) invoked with 2 parameters, 1 required\.$~'
         # for tsrc/Wizard.php
         - '~^Access to an undefined property Atk4\\Ui\\Wizard\:\:\$stepTemplate\.$~'
         - '~^Access to an undefined property Atk4\\Ui\\Wizard\:\:\$buttonFinish\.$~'
         # for tests/CallbackTest.php
-        - '~^Access to an undefined property Atk4\\Ui\\Tests\\AppMock\:\:\$terminate\.$~'
+        - '~^Access to an undefined property Atk4\\Ui\\tests\\AppMock\:\:\$terminate\.$~'
```
Comment (mvorisek) [inline phpstan.neon.dist]:
```suggestion
        - '~^Access to an undefined property Atk4\\Ui\\Tests\\AppMock\:\:\$terminate\.$~'
```

File: phpstan.neon.dist [hunk 5]
```diff
@@ -53,51 +38,21 @@ parameters:
...
-        # for demos/collection/lister-ipp.php
-        # for demos/collection/multitable.php
-        - '~^Method Atk4\\Ui\\Jquery\:\:addClass\(\) invoked with 1 parameter, 0 required\.$~'
-        # for demos/collection/table.php
-        - '~^Call to an undefined method Atk4\\Ui\\Jquery\:\:reload\(\)\.$~'
-        # for demos/collection/table2.php
-        - '~^Call to method getId\(\) on an unknown class Atk4\\Ui\\Model\.$~'
         # for demos/collection/tablefilter.php
-        - '~^Call to an undefined method Atk4\\Ui\\Demos\\CountryLock\:\:expr\(\)\.$~'
+        - '~^Call to an undefined method Atk4\\Ui\\demo\\CountryLock\:\:expr\(\)\.$~'
```
Comment (mvorisek) [inline phpstan.neon.dist]:
```suggestion
        - '~^Call to an undefined method Atk4\\Ui\\Demos\\CountryLock\:\:expr\(\)\.$~'
```

**Pull request merged**